components/login/login_mfa.tsx

```diff
@@ -32,10 +33,10 @@ const LoginMfa = ({loginId, password, title, subtitle, onSubmit}: LoginMfaProps)
     const handleSaveButtonOnClick = (e: React.MouseEvent | React.KeyboardEvent) => {
         e.preventDefault();
 
-        if (!saving) {
+        if (!saving && loginId) {
```

Comment:
I'm thinking we may want to allow the submit, and if `loginId` is `null` pass `loginId: ''`. Otherwise, in the case that there is no login id, they have no way to set it, and no way to know it isn't set. If we pass empty string, we can preserve empty behavior that they get an error back from the server when they click the button and have slightly more idea what's going on.

Reply:
Good point thanks. Empty string is passed in case of null